test/instrumentation/rack_instrumented_request_test.rb

```diff
@@ -29,17 +43,16 @@ def test_incoming_context
     expected = {
       trace_id: id,
       span_id: id,
-      from_w3: false,
+      from_w3c: false,
       level: '1'
     }
 
     assert_equal expected, req.incoming_context
     refute req.continuing_from_trace_parent?
   end
 
-  def test_incoming_w3_content
+  def test_incoming_w3c_context
     req = Instana::InstrumentedRequest.new(
-      'HTTP_X_INSTANA_L' => '1',
```

Comment:
I removed `x-instana-l` from all w3c-only tests, as this combination does not make sense.